docs/sphinx/source/whatsnew/v0.7.1.rst

```diff
@@ -0,0 +1,13 @@
+.. _whatsnew_0710:
+
+v0.7.1 (JULY 8, 2019)
+---------------------
+
+This is a minor release that does the following:
+
+**Simplify and optimize the performance the merge method in bifacial.py

```

Comment:
this should be in 0.7.0, right?

Reply:
My mistake, I thought 7.0 was already released. I'll correct this.

Reply:
@wholmgren I think the what's new file is all set now. I don't quite understand what about it caused the LGTM to fail. I think this PR is good to go.